packages/data/walkthroughs/9.9.9.json

```diff
@@ -1555,8 +1555,230 @@
+            {
+              "nextLogicType": "contains",
+              "answerToCheck": "P8",
+              "answerValue": "true"
+            }
+          ],
+          "nextNavigateTo": "R1.0"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "R2.0"
+        }
+      ]
+    },
+    "P20": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Does the egress door of this dwelling unit open onto an exit stair, public corridor, exterior passageway, or balcony, that serves more than one suite?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P20",
+          "answerValue": "false",
+          "nextNavigateTo": "R1.0"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P21"
+        }
+      ]
+    },
+    "P21": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "The egress door of the dwelling unit opens onto ...",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "An exit stairway",
+          "answerValue": "A"
+        },
+        {
+          "answerDisplayText": "A public corridor",
+          "answerValue": "B"
+        },
+        {
+          "answerDisplayText": "An exterior passageway",
+          "answerValue": "C"
+        },
+        {
+          "answerDisplayText": "A balcony",
+          "answerValue": "D"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P21",
+          "answerValue": "A",
+          "nextNavigateTo": "R2.0"
+        },
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P21",
+          "answerValue": "B",
+          "nextNavigateTo": "P22"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P23"
+        }
+      ]
+    },
+    "P22": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Is the public corridor served by a single exit?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P22",
+          "answerValue": "false",
+          "nextNavigateTo": "R1.0"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "R2.0"
+        }
+      ]
+    },
+    "P23": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Is the {answer from P21} served by a single exit stairway or ramp?",
```

Comment:
Note that @nicholaslyon-dt will have to decide a way to handle the variable in this question text.

Reply:
Yeah I'd probably swap this with a custom tag with a data prop, something like `<answer-value answer="P21" />` and then add an extra property to the P21 answers like "answerValueDisplay": "exterior passageway".